CHANGELOG.md

```diff
@@ -1,46 +1,331 @@
-## [Unreleased][unreleased]
+## [0.11.0] - 2017/08/16
+
+The latest and greatest version of Kong features improvements all over the
+board for a better and easier integration with your infrastructure!
+
+The highlights of this release are:
+
+- Support for **regex URIs** in routing, one of the oldest required
```

Comment:
ah, I thought the same when I read this but didn't think it was worth mentioning at first. Since @salazar noticed it as well, I +1 on this.